### ISSUE TITLE
how to find the schema.name?

### ISSUE THREAD

**Issue by vivk89** [DataReply/kafka-connect-mongodb#19]

**Comment by xuelvming** [comment]
Yep, I have the same question, could you please provide the info ,what is schema.name.where to find it in my current mongodb collection? 
Thanks

**Comment by zoeminghong** [comment]
+1

**Comment by zoeminghong** [comment]
@xuelvming @vivk89 The schema.name can write by timestamp,order, operation, database or object.
[https://github.com/DataReply/kafka-connect-mongodb](https://github.com/DataReply/kafka-connect-mongodb)
